setup.py

```diff
@@ -4,7 +4,7 @@
 
 
 setup(
-    name='statsd',
+    name='statsd-telegraf',
```

Comment:
Actually, this can be reverted. I was changing the name so we could set this as a dependency so when working on jaycee/pyramid_metrics_tags it wouldn't fetch statsd from pypi accidentally. But it turns out there's also a statsd-telegraf on pypi (which doesn't solve our problem) that get's pulled in, so I had to make changes to pyramid_metrics anyway.